### ISSUE TITLE
youtube and other entertainment streaming sites block

### ISSUE THREAD

**Issue by isaacn75** [StevenBlack/hosts#1140]
hi , im ussing pfsense with pfblock to do dns block on some sites , i want to block streaming sites like youtube ,netflix , hulu . etc . is there a host file for those sites ? cant find it .

**Comment by StevenBlack** [comment]
Hi @isaacn75, interesting idea.  We don't have a streaming/entertainment extension.  Maybe we should.

**Comment by ScriptTiger** [comment]
This one may actually be rather easy to implement without too many extra entries just by shutting down the CDNs, then you don't even have to really worry about the individual website front ends.

**Comment by FahadBinHussain** [comment]
Yes, this is an essential feature that has been in high demand for quite some time.

**Comment by theking2** [comment]
> This one may actually be rather easy to implement without too many extra entries just by shutting down the CDNs, then you don't even have to really worry about the individual website front ends.


Not sure if shutting CDN is a good thing. They make the Internet and many sites work (faster).

**Comment by ScriptTiger** [comment]
@theking2, are you bringing this up because you're interested in reviving this extension proposal? It's definitely an interesting proposal, so whatever suggestions you have are definitely welcomed. I started a repo some years ago to play with it, but never really went anywhere with it and it's now archived. There's definitely interest out there for it, but just not enough for any movement behind it, unfortunately. So, if you'd like to pick things up and spitball further ideas, I'm sure we'd all be opened to revisit this and discuss things further. I look forward to hearing your thoughts.

**Comment by theking2** [comment]
Ah ok. Sorry. I will leave this here
